Python/Product/Common/Infrastructure/CancellationTokens.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Microsoft.PythonTools.Infrastructure {
     public static class CancellationTokens {
```

Comment:
I'm confused about how this entire class is supposed to work. Aren't all these tokens essentially single-use for the lifetime of the process they're created in? And won't they start counting from the moment they're created?